test/testapps/BasicTestApp/.gitignore

```diff
@@ -0,0 +1,2 @@
+wwwroot/js/**/*.js
+wwwroot/js/**/*.js.map
```

Comment:
If we specified a `<TypeScriptOutFile>` in the csproj, would that stop it from generating all these `.js` and `.js.map` files? It would be great not to have all those files appearing (and hence we wouldn't need these lines in `.gitignore` either)

Reply:
Yeah, this needs cleanup. I'll webpackify all these things

Reply:
Do we need to use TypeScript here in `BasicTestApp` though? As per comment below I'm not sure it's advantageous.